src/test/java/integration/AuthenticationIntegrationTest.java

```diff
@@ -0,0 +1,38 @@
+package integration;
+
+import io.dropwizard.testing.junit5.DropwizardAppExtension;
+import io.dropwizard.testing.junit5.DropwizardExtensionsSupport;
+import org.example.TestApplication;
+import org.example.TestConfiguration;
+import org.example.models.LoginRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+
+@ExtendWith(DropwizardExtensionsSupport.class)
+public class AuthenticationIntegrationTest {
+    private static final DropwizardAppExtension<TestConfiguration> APP =
+            new DropwizardAppExtension<>(TestApplication.class);
+
+    @Test
+    void loginShouldReturnJWTToken() {
+        Client client = APP.client();
+
+        LoginRequest loginRequest = new LoginRequest("admin", "admin");
```

Comment:
don't hardcode valid login credentials in code, instead pass them in as environment variables